template.html

```diff
@@ -71,7 +71,7 @@ <h2><a href="#infi-test">
             {{#each test.checks}}
             <li class="topic-{{topic}}">
               <div class="topic-icon {{topic}}">
-                <img src="/resources/{{topic}}.svg" alt="icon of {{topic}}" type="image/svg+xml">
+                <a href=#{{topic}}><img src="/resources/{{topic}}.svg" alt="icon of {{topic}}" type="image/svg+xml"></a>
               </div>
               <span>{{question}}</span>
```

Comment:
I was wondering... Would it be nice if we could just click the whole line (icon + text) to jump towards that section, or why did you choose for the icon as a link specifically?